### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by Geller99**
- Built calendar component with moment js
- User can access current date by clicking on calendar date
- Still in progress

### CODE REVIEW

**Review by royboy789 — COMMENTED** [review]
Can you remove `DesignPrinciples.md` and the accordion component updates from this PR?

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[0210/154535.755:ERROR:registration_protocol_win.cc(102)] CreateFile: The system cannot find the file specified. (0x2)
```
Comment (ashishkrtewari) [inline debug.log:1]: Let's add this file to .gitignore

File: src/packages/calendar/index.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
+export {}
```
Comment (ashishkrtewari) [inline src/packages/calendar/index.ts:2]: Why this file, if we are not exporting anything?

File: src/packages/calendar/src/Calendar.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    <div className="New Div">
+      {calendar.map((week: any) => {
+        return (
+          <Day>
+            {week.map((day: any) => {
+              return (
+                <div
+                  onClick={() => {
+                    setInitialDate(day);
+
```
Comment (ashishkrtewari) [inline src/packages/calendar/src/Calendar.tsx:42]: unnecessary line break

File: src/packages/calendar/src/useCaseCalendar.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+
+
+const UseCaseCalendar = () => {
+
+  return (
+    <div>
```
Comment (ashishkrtewari) [inline src/packages/calendar/src/useCaseCalendar.tsx:6]: the use case file should have the Calendar component implementation?

**Pull request merged**